starknet_py/cairo/type_parser.py

```diff
@@ -86,6 +85,7 @@ def _transform_cairo_lang_type(
                     OrderedDict(
                         (member.name, self._transform_cairo_lang_type(member.typ))
                         for member in cairo_type.members
+                        if member.name is not None  # just for typechecker
```

Comment:
Hmm, right now the members will be filtered and a member without name just won't be included. Doesn't a member without a name mean that something went wrong? Would asserting that all names are present before the return be any better?